src/Integration/Notifications/NotificationIndicator.xaml

```diff
@@ -1,20 +1,27 @@
 ﻿<UserControl x:Name="UserControl" x:Class="SonarLint.VisualStudio.Integration.Notifications.NotificationIndicator"
         xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-        xmlns:local="clr-namespace:SonarLint.VisualStudio.Integration.Notifications"
         xmlns:converters="clr-namespace:SonarLint.VisualStudio.Core.WPF;assembly=SonarLint.VisualStudio.Core">
     <UserControl.Resources>
         <converters:BoolToVisibilityConverter x:Key="BoolToVisibilityConverter" />
         <converters:BoolToVisibilityConverter x:Key="InvertedBoolToVisibilityConverter" TrueValue="Collapsed" FalseValue="Visible" />
+        <SolidColorBrush x:Key="LogoBlueBrush" Color="#D5D5D5" />
+        <DrawingImage x:Key="SonarQubeIconDrawingImage" >
```

Comment:
The server notifications refer to both Server and Cloud. However, here we use the SonarQube Server icon. Previously this was the case because we didn't support properly the SonarCloud. It would be worth checking if we can fix that, maybe by using the SonarQube for IDE icon.

Reply:
I created a time-boxed sub-task for showing it conditionally:
https://sonarsource.atlassian.net/browse/SLVS-1607